src/content/pages/thank-you.mdx

```diff
@@ -0,0 +1,273 @@
+
+Huge thanks to **Niklas Mertsch** for leading Discord and bot management!
+
+The work on maintaining and updating the discord bot can be found on the [EuroPython discord bot repo](https://github.com/EuroPython/discord).
+
+
+## A/V Team
+Although an external supplier handles streaming and recording, this team communicates with the venue, manages digital screens, and handles unforeseen technical issues.
+
+Big thanks to **Raquel Dou** and **Piotr Gnus** for A/V coordination.
+
+
+## Onsite Volunteers Team
+
+Once you enter the conference, the first what you see are people in yellow t-shirts. They're our volunteers, who assist attendees, chair sessions, and hand out materials.
+The onsite volunteers sub-team manages volunteer recruitment and scheduling:
+- xxx
+
+
+# Onsite & Remote Volunteers
+Volunteers visible at the conference in yellow shirts, and those supporting remotely:
+
+## Onsite Volunteers
+- xxx
+
+## Remote Volunteers
+- xxx
+
+# Events & Workshops Organisers
+Beyond talks and tutorials, EuroPython includes engaging events and workshops.
+
+## Django Girls
+A workshop designed to inspire less represented groups, particularly women, to get interested in technology. Big thanks to Doreen Peace Nangira Wanyama for organizing!
+
+
+## Humble Data
+
+Data workshop designed for total beginners coming from less represented groups. Organizers and mentors:
+
+- Cheuk Ting Ho
+- Jodie Burchell
+- Martin Borus
+- Sena Şahin
+
+Special thanks to **Jodie Burchell** for coordinating beginner workshops.
+
+## Unconference Day
+In parallel with sprints and workshops, there was an unconference day featuring Q&A, discussions, and unconference-style sessions, inspired by the same concept hosted at PyCon CZ 23. Huge thanks to organizers **Daria Grudzien Linhart** and **Honza Javorek**.
+
+## PyLadies Events
+On Thursday, PyLadies hosted multiple events for PyLadies and allies to get together. Shoutout to **Maria-José Molina** for coordination.
+
+## Sprints
+If you came to sprints, you could have seen open-source maintainers and people submitting PRs to projects. Many thanks to **Rodrigo Girão Serrão** for organizing sprints and supporting all maintainers-spring organizers.
+
+# Code of Conduct Committee
+
+We have a code of conduct in place to make sure that the conference is a safe place for many different people. This team is responsible for receiving and handling CoC reports:
+
+- Cheuk Ting Ho
+- Jakub Vysoky
+- Lumír Balhar
+- Vicky Twomey-Lee
+
+----
+
+# Acknowledgement
+
+Apart from volunteers, we also have a few trusted suppliers we work with.
+
+
+## Adriana Sánchez - Design
+
+Huge thanks to **Adriana Sánchez** for her excellent work on digital and printed design materials!
+
+
+## Gonzo - AV Team
+
+Big thanks to our AV team Gonzo Design who have been streaming, recording & editing our talks, and more importantly, staying with us since 2017! Here are the folks who helped us with streaming and the smooth hybrid experience:
```

Comment:
Maybe link to the Gonzo website?

Reply:
It's https://www.gonzodesign.it/ if you want to put it up.